core/config/project_settings.cpp

```diff
@@ -291,31 +291,26 @@ bool ProjectSettings::_set(const StringName &p_name, const Variant &p_value) {
 			return true;
 		}
 
-		if (!disable_feature_overrides) {
+		{ // Feature overrides.
 			int dot = p_name.operator String().find(".");
 			if (dot != -1) {
 				Vector<String> s = p_name.operator String().split(".");
 
-				bool override_valid = false;
 				for (int i = 1; i < s.size(); i++) {
 					String feature = s[i].strip_edges();
```

Comment:
Reading this more in depth for the first time - it supports more than one feature override for a given property name? i.e. `keep_screen_on.macos.windows` works to override for both `macos` and `windows`?

I'm not sure we ever use or test this if so, which might mean it could be buggy in some places.

Reply:
No idea, it seems it always did, I don't mind leaving it there since there is no harm I guess. It should definitely not be buggy, though since it relies on `split()` for everything.